src/app/features/issue/issue.model.ts

```diff
@@ -85,32 +85,32 @@ export type IssueData =
   | RedmineIssue;
 
 export type IssueDataReduced =
-  | GithubIssueReduced
-  | JiraIssueReduced
-  | GitlabIssue
-  | OpenProjectWorkPackageReduced
-  | CaldavIssueReduced
-  | ICalIssueReduced
-  | GiteaIssue
-  | RedmineIssue;
+  | (GithubIssueReduced & { plannedAt?: string | null })
+  | (JiraIssueReduced & { plannedAt?: string | null })
+  | (GitlabIssue & { plannedAt?: string | null })
+  | (OpenProjectWorkPackageReduced & { plannedAt?: string | null })
+  | (CaldavIssueReduced & { plannedAt?: string | null })
+  | (ICalIssueReduced & { plannedAt?: string | null })
+  | (GiteaIssue & { plannedAt?: string | null })
+  | (RedmineIssue & { plannedAt?: string | null });
 
 export type IssueDataReducedMap = {
   [K in IssueProviderKey]: K extends 'JIRA'
-    ? JiraIssueReduced
+    ? JiraIssueReduced & { plannedAt?: string | null }
     : K extends 'GITHUB'
-      ? GithubIssueReduced
+      ? GithubIssueReduced & { plannedAt?: string | null }
```

Comment:
That doesn't look right. You shouldn't have to change the other issue provider issue types.